docs-sphinx/prepare_docstrings.py

```diff
@@ -328,10 +328,8 @@ def handle_function(functionname, cls):
     handle_module(modulename, module)
     if module.__file__.endswith("__init__.py") and modulename != "uproot":
         for submodulename in sorted(
-            [
-                modulename + "." + name
-                for loader, name, is_pkg in pkgutil.walk_packages(module.__path__)
-            ]
+            modulename + "." + name
+            for loader, name, is_pkg in pkgutil.walk_packages(module.__path__)
```

Comment:
These sorts of fixes are not really always python upgrade syntax related, but welcome. It no longer has to construct an extra temporary list before running the sort, and removes brackets too, a win-win. ;)

Reply:
I'd actually prefer `f"{modulename}.{name}"`, faster and I think it reads better than using a +, but okay.

Reply:
I only use `+` for short things. There's a threshold for "short" that gets reevaluated now that the f-string syntax is available. `f"{modulename}.{name}"` > `modulename + "." + name` > `"{0}.{1}".format(modulename, name)`.

Reply:
Yes, not making statements on the old versions of things (I'd use + over format for something like this), but pointing out exactly this change (plus f-strings are wickedly fast for things like this, no extra function call(s)).